packages/components/src/Account/Auth/AuthStyledComponents.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from "react";
+import styled, { StyledComponentClass } from "styled-components";
+import { colors, fonts } from "../../styleConstants";
+import * as checkEmailImage from "../../images/auth/img-check-email@2x.png";
+import { PageHeadingTextCenteredSmall } from "../../Heading";
+import { Link } from "react-router-dom";
+
+export const CheckboxContainer = styled.ul`
+  list-style: none;
+  padding-left: 0;
+  margin-top: 0;
+`;
+
+export const CheckboxSection = styled.li`
+  margin-bottom: 10px;
+`;
+
+export const CheckboxLabel = styled.span`
+  color: ${colors.primary.CIVIL_GRAY_1};
+  font: 400 12px/20px ${fonts.SANS_SERIF};
+  padding-left: 7px;
+`;
+
+export const ConfirmButtonContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  padding-top: 20px;
+`;
+
+export const CheckEmailLetterIcon = styled.div`
+  width: 108px;
+  height: 108px;
+  background-image: url(${checkEmailImage});
+  background-size: cover;
+  margin: 30px 0;
+`;
+
+export const CenterWrapper: React.SFC = ({ children }) => (
+  <div style={{ display: "flex", justifyContent: "center" }}>
+    <div>{children}</div>
+  </div>
+);
+
+export const CheckEmailWrapper = styled.ul``;
```

Comment:
Is this being used? If there are no styles on this you should just use ul.

Reply:
Nope, good catch!